.github/workflows/build.yml

```diff
@@ -52,13 +52,17 @@ jobs:
       # Please declare any key you added below in build_steps.yaml:jobs.compile.strategy.matrix to aid the linting tools
       # Please update publish.yml if changing the symbols value!!
       linux_matrix: >-
-        {"os":"linux", "distro":"\"ubuntu-latest\"", "pytest_distro": "${{vars.PYTEST_DISTRO_LINUX}}",
```

Comment:
Ah, this must have been confusing. I use it to divert my fork to run tests on my home cluster, so it's possible to debug the test job.

Let's remove it for now. I'll properly introduce it if we host our own runners.